cmd/remove-members/test/members.txt

```diff
@@ -0,0 +1,2 @@
+johndoe
```

Comment:
another test case to consider: a file not named `orgs.yaml` or `teams.yaml` that has a member's name in it